package.json

```diff
@@ -21,6 +21,7 @@
     "@babel/plugin-proposal-class-properties": "7.7.4",
     "@babel/plugin-proposal-optional-chaining": "^7.16.7",
     "@babel/plugin-transform-react-jsx-source": "7.7.4",
+    "@babel/plugin-proposal-nullish-coalescing-operator": "7.17.12",
```

Comment:
Have you ran `yarn install` after adding in `package.json`?

Reply:
Oh, I wasn't aware of that, are there any more changes, I'll push all of them together in a single commit.

Reply:
done